core/commands/dht.go

```diff
@@ -581,3 +597,18 @@ PutValue will store the given key value pair in the dht.
 	},
 	Type: notif.QueryEvent{},
 }
+
+func escapeDhtKey(s string) (key.Key, error) {
+	if s[0] != '@' {
+		return key.B58KeyDecode(s), nil
+	}
```

Comment:
I remember us making an explicit choice to not base58 encode the dht keys, i think due to how horrendously slow it is to encode and decode them.